src/meta/mod.rs

```diff
@@ -56,8 +56,14 @@ impl Meta {
             println!("cannot access '{}': {}", path.display(), err);
             return Ok(meta);
         }
+		let mut content = Vec::new();
+
+		let current_meta = Self::from_path(&PathBuf::from("."))?;
```

Comment:
We should probably add the path that is received as just checking for `.` will not take into consideration the path received 

as in if someone gives

```
lsd ~/Documents
```

we should be providing the `.` and `..` relative to `~/Documents` and not the folder the user in.